attending/library.py

```diff
@@ -77,17 +78,17 @@ def get_edition(self, name, version):
         return Module(self.location, name)[version]
 
 
-def attending_doc(module, version=None):
+def attending_doc(module, version=None, home=Path().home()):
```

Comment:
I forget, are Path objects mutable?

Reply:
Paths are immutable and hashable (so you can use them as keys in a dictionary, for example). https://docs.python.org/3/library/pathlib.html#general-properties